Dockerfile

```diff
@@ -11,6 +11,8 @@ RUN dnf -y update && dnf install -y make git golang golang-github-cpuguy83-go-md
 	# OpenShift deps
 	which tar wget hostname util-linux bsdtar socat ethtool device-mapper iptables tree findutils nmap-ncat e2fsprogs xfsprogs lsof docker iproute \
         bats jq podman \
+	# golint from packages instead of upstream
```

Comment:
This is not quite a meaningful comment at this point; what’s the reader supposed to do or not do based on it?

Maybe just the commit message is enough. Or this should say something like ”don’t use (go get) to install this because…”, and it’s not clear to me why it would apply specially to `golint` and not to other packages like `podman` just above.

Reply:
I'll remove the line to avoid confusion.